### DISCUSSION TITLE
Is capacitor leakage a concern?

### DISCUSSION THREAD

**Discussion by GeorgeHahn** [joeycastillo/Sensor-Watch#478] [General]
Hi,

I'm curious if capacitor leakage current is important for this design. I expected it would be, but my interpretation of an app note on the subject[^1] is that leakage is relatively minor in circuits that stay energized over a long duration. Does that line up with what you've seen with this design?

[^1]: [Silicon Labs: Current Leakage through Ceramic
Capacitors [PDF]](https://www.silabs.com/documents/public/application-notes/an1415-current_leakage_through_ceramic_capacitors.pdf)
[^2]:

**Comment by joeycastillo** [comment]
First that was a fascinating document! And second it does line up with my experience. Most of the decoupling capacitors on the 3v3 power rail are 0.1 uF; there’s only one 10uF cap, and I experimented there with different dielectrics and even flirted with making it 4.7 uF. In my testing, even measuring at the nA range, I wasn’t able to notice a significant difference, either while operating the watch or in low energy mode. 

I did switch the 10uF from an X5R to an X7R capacitor between the original board and Sensor Watch Lite (which this document suggests is slightly worse) but in the end, improvements in the software led to the much bigger wins in terms of power consumption. 

Also as a side note, one of the reasons that testing this is so subtle is that at these low current consumption scenarios, just breathing on the test setup can affect your results. Like in one test, the watch was running at a steady draw of ~6uA at room temperature, but heating it up closer to body temperature, power consumption grew to -9uA. In this scenario you can imagine how a few dozen nA in leakage current between a 4.7 and a 10 uF capacitor could get lost in the noise.
